docs/changelog.md

```diff
@@ -55,6 +55,15 @@ permalink: /changelog/
 * **feature-prompts**
   * Adds support for Web Share API using `ShareDelegate`.
 
+* **lib-dataprotect**
+  * Added new `SecureAbove22Preferences` helper class, which is an encryption-aware wrapper for `SharedPreferences`. Only actually encrypts stored values when running on API23+.
+
```

Comment:
What's our plan to QA this? This seems like something that will be rather painful if it doesn't work on some configurations. Will be do that in Fenix once the actual feature is available?